src/scripts/Selector/SelectorTable.js

```diff
@@ -65,22 +65,20 @@ export default class SelectorTable extends Selector {
 		result = Array.from({
 			length: dataCells.length / Object.keys(columnIndices).length,
 		}).map(_ => Object());
+		const offset = dataCells[0].cellIndex; // cellIndex = порядковому номеру элемента среди его одноуровневых элементов
```

Comment:
Again, that comment is redundant.
cellIndex is a [standard table cell property](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableCellElement)
Instead you may give this variable more explanatory name, like `columnOffset` or even `firstDataColumnOffset`.